hw/ip/entropy_src/doc/programmers_guide.md

```diff
@@ -52,6 +52,13 @@ If firmware wants to use hardware conditioning for the inserted entropy bits, it
 1. Set the `FIPS_ENABLE` field in the [`CONF`](registers.md#conf) register to `kMultiBitBool4True`.
 1. Set `ES_TYPE` or `ES_ROUTE` (or both) in [`ENTROPY_CONTROL`](registers.md#entropy_control) to `kMultiBitBool4False`.
 
+Otherwise, if firmware does not want to use hardware conditioning for the inserted entropy bits, it has to do one of the two following points:
+1. Set the `FIPS_ENABLE` field in the [`CONF`](registers.md#conf) register to `kMultiBitBool4False`.
+1. Set `ES_TYPE` and `ES_ROUTE` in [`ENTROPY_CONTROL`](registers.md#entropy_control) to `kMultiBitBool4True`.
+
+For the first case, the entropy will be routed to software.
+For the second case on the other hand, the entropy will be routed to the hardware endpoints.
+
 It further has to set the `FW_OV_INSERT_START` field in the [`FW_OV_SHA3_START`](registers.md#fw_ov_sha3_start) register to `kMultiBitBool4True`.
```

Comment:
This paragraph is related to the conditioning again. So maybe it's better to add your newly added paragraph to the very end of this section.

Reply:
I moved it towards the end of the section now